shell/platform/darwin/macos/framework/Source/FlutterView.mm

```diff
@@ -15,6 +16,7 @@ @interface FlutterView () <FlutterResizeSynchronizerDelegate> {
   __weak id<FlutterViewReshapeListener> _reshapeListener;
   FlutterResizeSynchronizer* _resizeSynchronizer;
   FlutterSurfaceManager* _surfaceManager;
+  bool _embedded_views_preview_enabled;
```

Comment:
Don't need this ivar, can just use [FlutterPlatformViewController embeddedViewsEnabled] when necessary.

Reply:
ivars should be camelCase.